packages/mui-joy/src/styles/extendTheme.ts

```diff
@@ -108,8 +108,8 @@ export default function extendTheme(themeOptions?: CssVarsThemeOptions): Theme {
     success: colors.green,
     warning: colors.yellow,
     common: {
-      white: '#FCFCFD',
-      black: '#09090B',
+      white: '#fff',
+      black: '#000',
     },
```

Comment:
Maybe it's a matter of either creating a new token or updating the templates (or both), but I think that, probably this changes, inverted things for the templates a bit. Like, in all of them, the side nav turned into neutral 50 and the background to plain white, which I don't really think makes sense ⎯ the previous version was more correct in my perspective (i.e. darker colors being more behind vs. lighter colors being higher).

This seems more of a thing for light mode, though ⎯ dark looks alright!

<img width="500" alt="Screen Shot 2023-08-14 at 19 36 59" src="https://github.com/mui/material-ui/assets/67129314/8a09ad52-b401-4240-b78b-18bd3a5d952d">

Reply:
It was on purpose, and it's actually [here](https://github.com/mui/material-ui/pull/38416/files/db9cb9d3e81d68a8cdebde63e947568d0ec85ce9#diff-158304f3a4b17f65c8bd9dd688681ae17e38aa6da3cabaaa8bbaae0a41948710L219). 
I was checking different apps and websites using the light mode, and almost all of them had the body color as plain white and side navs, menus, cards, etc were tones of gray.
Although the previous approach had this logic behind, to create depth, [this comment](https://www.notion.so/Olivier-design-review-on-Joy-Design-3ada9a7bcfa44b9fab1fe5032dfb33bb?d=82ec1ea8864a42a0b0155a6ca8f7fbdd&pvs=4#c707c9d4c0f5424f870b2fd071911358) from @oliviertassinari  standpoint made me think if we really want to go with this logic, or if we intend to make something that's more common out there.

Reply:
1. This is great, a bug fix, common was created to not repeat #fff everywhere in the codebase, so anything else than #fff or #000 didn't make sense.

```diff
    common: {
-     white: '#FCFCFD',
-     black: '#09090B',
+     white: '#fff',
+     black: '#000',
    },
```

2. Material Design has darker color above lighter ones, e.g. https://m3.material.io/components/navigation-rail/specs so I think it's a matter of perspective and UI was are used to. 

3. Personally, I would go with everything white in light mode, what Material Design used to do, it would be refreshing if Material UI go with background colors too.

Reply:
> 1. To be fair, I'm not even sure we need `common`

I think so. We can remove `common` from the default theme. I don't find it useful, to be honest.

Reply:
> 1. I think so. We can remove `common` from the default theme. I don't find it useful, to be honest.
I agree, if we end up by using only #000 and #FFF, I think we don't need to have the common black and white.